dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -348,10 +351,11 @@ binaryRoundtrip expression =
         -> Either DeserialiseFailureWithEq a
     wrap = Data.Coerce.coerce
 
--- isNormalizedIsConsistentWithNormalize :: Expr () Import -> Property
--- isNormalizedIsConsistentWithNormalize expression =
---         Dhall.Core.isNormalized expression
---     === (Dhall.Core.normalize expression == expression)
+isNormalizedIsConsistentWithNormalize :: Expr () Import -> Property
+isNormalizedIsConsistentWithNormalize expression =
+    case Control.Spoon.spoon (Dhall.Core.normalize expression) of
+        Just nf -> Dhall.Core.isNormalized expression === (nf == expression)
+        Nothing -> Test.QuickCheck.discard
```

Comment:
I have updated the test to simply discard if `normalize` returns an error. I'd argue that this covers the covers the cases that we care about the most.